### ISSUE TITLE
Plugin doesn't load Images into Krita anymore. Images are generated though (checked temporary files)

### ISSUE THREAD

**Issue by AllreadyTaken** [sddebz/stable-diffusion-krita-plugin#25]
Hey :)
Thanks for your amazing work with this plugin. Sadly I discovered that by today when starting up Krita and the Server, the Images are not loaded into krita anymore. The Server gets all the information from Krita, triggers the Post Request but inside Kriuta the new layers come in empty. I checked that nothing is off with stable diffusion by disabling the automated deletion of temporary images.  Neither text2img nor img2img works. The nwe layers created in Krita have no content and are transparent.
Is there a way to debug this? / What could be the issue?

Edit: Logger Inspection of Krita shows:
![image](https://user-images.githubusercontent.com/12010863/189479662-2e4bbad5-aa79-4ff2-a812-485e65977624.png)

**Comment by AllreadyTaken** [comment]
mmh the krita logger seems to not showing any print statement from the  Script Methods whatsoever. 
Tried also to run Krita in Admin Mode , but the results are the same. Rather frustrating since i worked perfectly yesterday :-/

**Comment by AllreadyTaken** [comment]
ok opened krita trough mingw. cleared the krita QT cache. wrote some print statements into insert_img function of krita_diff.py
ba variable is not empty. general print statements don't show up in minGw except ba which prints the whole content as a binary.

BTW:
Krita Version: 5.1.0, Qt version compiled: 5.12.12, loaded: 5.12.12. 

so i give up for now.just try to reboot.

**Comment by AllreadyTaken** [comment]
Solved!
I leave this ticket here as a symbol for my idiocy. 
After dismantling the script and putting the image loading function back in line by line to be able to debug it and just saw that everything should work. 
I discovered : I changed the Pixel Depth from 8 bit integer to 16 bit integer for newly created documents late last night. 

Thise lead to the Stable Diffusion Images not having enough Data to be imported

So 8bit Pixel Depth is mandatory for this Plugin to run. 

Hooray.
